live-usb-creator/live_scripts/startup

```diff
@@ -3,9 +3,11 @@
 # /etc/rc.d/init.d/livesys calls out to this script (/usr/local/bin/startup), meaning it’s run on startup.
 # Note that it does NOT block a TTY/login/bash/etc., hence the nfast_block_shell utility script.
 
-/usr/local/bin/install_protoc
+/usr/local/bin/install_protobuf
 
 /usr/local/bin/install_nfast_tools
 
+ln -Tsf /bin/python3 /bin/python
```

Comment:
Will python3 be the only installed version? Or, does this overwrite a 2.x default?

Reply:
This overrides the python2 default.